android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageBase.java

```diff
@@ -360,11 +359,8 @@ protected String decryptBytes(@NonNull final Key key, @NonNull final byte[] byte
         handler.initialize(cipher, key, in);
```

Comment:
```suggestion
        handler.initialize(cipher, key, bytes);
```

Now that the whole decrypt process works with byte arrays, it might be better to change the `DecryptBytesHandler` to take the input as a byte array, instead of an input stream.

This change also requires the `DecryptBytesHandler` interface input parameter to be changed to `byte[]`, but shouldn't require other changes as the parameter is only passed to overloaded methods which already handle byte arrays as well as InputStreams.

Reply:
👍 This was a miss. I should've cleaned this up.